smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/ProcessorMediator.java

```diff
@@ -30,6 +32,17 @@ public ProcessorMediator(MediatorConfiguration configuration) {
         if (configuration.shape() != Shape.PROCESSOR) {
             throw ex.illegalArgumentForProcessorShape(configuration.shape());
         }
+
+        // IMPORTANT When returning a Publisher or a PublisherBuilder, you can't mix payloads and messages,
```

Comment:
Does this also apply to `Multi`?

Reply:
Yes.

Reply:
Asking because the exception messages don't mention `Multi` and `InvalidProcessorSignatureTest` doesn't test `Multi` (and `InvalidStreamTransformerSignatureTest` doesn't test `Publisher`).

I'd _guess_ that the answer is yes, but I'm not really sure :-)

Reply:
Haha, got an answer sooner than I added a clarification :-) Thanks!

(That said, my questions in PR reviews are often rhetorical. The comment should say this IMHO.)

Reply:
I've extended the comment and the failure messages.

Reply:
I know I can be demanging :-), thanks!

Reply:
It needed to be done! I wrote the comment before the code.